classes/PodsAdmin.php

```diff
@@ -2331,29 +2331,37 @@ public function admin_capabilities ( $capabilities ) {
                     $capabilities[] = 'delete_' . $capability_type;
 
                     if ( 1 == pods_var( 'capability_type_extra', $pod[ 'options' ], 1 ) ) {
```

Comment:
Changes post_type code to be similar as what I've done for taxonomies. Consistency FTW!